content/en/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -147,3 +147,4 @@ request and limit, the same as the scheduler.
 * Read about [creating a Pod that has an init container](/docs/tasks/configure-pod-container/configure-pod-initialization/#create-a-pod-that-has-an-init-container).
 * Learn about the [types of probes](/docs/concepts/workloads/pods/pod-lifecycle/#types-of-probe): liveness, readiness, startup probe.
 * Learn about [pod overhead](/docs/concepts/scheduling-eviction/pod-overhead/).
+* Learn how to [Adopt Sidecar Containers](/docs/tutorials/configuration/pod-sidecar-containers/)
```

Comment:
I'd list this as the first item in the list, not the last/

Reply:
makes sense. moved